datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -74,7 +74,7 @@
 @Service
 public class SdxReactorFlowManager {
 
-    private static final long WAIT_FOR_ACCEPT = 10L;
+    private static final long WAIT_FOR_ACCEPT = 20L;
```

Comment:
Is it necessary?

Reply:
I think 10 sec should be enough to process any request. If it's not then we need to fix those endpoints. Increasing this timeout just hides the problem. What is your opinion @sodre90?

Reply:
If it can not be accepted in 10 sec then there are API calls where it should not be, please check it and don't modify it.

Reply:
can we find out why the related flow acceptance takes more than 10 sec? Do we have api calls in flow configuration or something like this? @kkalvagadda1

Reply:
reverted this change.